JustSaying/JustSayingFluently.cs

```diff
@@ -187,9 +187,20 @@ public ISubscriberIntoQueue WithSqsPointToPointSubscriber()
 
         #region Implementation of Queue Subscription
 
-        public IFluentSubscription IntoQueue(string queuename)
+        public IFluentSubscription IntoQueueNamed(string queueName)
         {
-            _subscriptionConfig.BaseQueueName = queuename;
+            if (string.IsNullOrWhiteSpace(queueName))
+            {
+                throw new ArgumentNullException(nameof(queueName));

```

Comment:
Hmm, I see what you are saying. Why not have ```IntoDefaultQueue``` be an optional alternative for ```.IntoQueue(string.Empty)``` and keep ```IntoQueue``` behaving how it does now.
In our usage we do the scenario you talk about, i.e. a variable that is sometimes ```string.Empty```